### PR TITLE
[Improvement](statistics)Add config for the threshold of column count for auto analyze.

### PR DESCRIPTION
**Pull request by Jibing-Li**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Kikyou1997 — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java [hunk 1]
```diff
@@ -178,6 +178,11 @@ protected AnalysisInfo getReAnalyzeRequiredPart(AnalysisInfo jobInfo) {
         AnalysisManager analysisManager = Env.getServingEnv().getAnalysisManager();
         TableStatsMeta tblStats = analysisManager.findTableStatsStatus(table.getId());
 
+        // Skip tables that are too width.
+        if (table.getColumns().size() > StatisticsUtil.getAutoAnalyzeTableWidthThreshold()) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java:182]: do we need add ut to check the Threshold work well?

**Comment by xiaokang** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java (hunk 1 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java]: Added

**Review by morningman — APPROVED** [review]

**Pull request merged**